relay-server/src/actors/spooler/mod.rs

```diff
@@ -1,3 +1,30 @@
+//! This module contains the [`BufferService`], which responsible for spooling of the incoming
+//! envelopes, either in memory or on persistent storage.
+//!
+//! The main entry point for the [`BufferService`] is the [`Buffer`] interface. Which currently
+//! supports:
+//! - [`Enqueue`] - enqueueing message into the backend storage
+//! - [`DequeueMany`] - dequeueing all the requested [`QueueKey`] keys
+//! - [`RemoveMany`] - Removing and dropping the requested [`QueueKey`] keys.
+//! - [`Health`] - checking the health of the [`BufferService`]
```

Comment:
```suggestion
//! - [`Enqueue`] - enqueueing a message into the backend storage.
//! - [`DequeueMany`] - dequeueing all the requested [`QueueKey`] keys.
//! - [`RemoveMany`] - removing and dropping the requested [`QueueKey`] keys.
//! - [`Health`] - checking the health of the [`BufferService`]
```
For `RemoveMany`, could we leave `removing`?
```
//! - [`RemoveMany`] - removing the requested [`QueueKey`] keys.
```